FinSearchUnified/ShopwareProcess.php

```diff
@@ -131,7 +122,7 @@ public function getAllProductsAsXmlArray($selectedLanguage = 'de_DE', $start = 0
             }
 
             /** @var FindologicArticleFactory $findologicArticleFactory */
-            $findologicArticle = $findologicArticleFactory->create($article, $this->shopKey, $allUserGroups, $articleSales, $baseCategory);
+            $findologicArticle = $findologicArticleFactory->create($article, $this->shopKey, $allUserGroups, [], $baseCategory);
```

Comment:
Please remove the `$salesFrequency` argument since it has become obsolete. Therefore we also have to postpone this pull request for a later release until we know how to handle backward compatibility with `ExtendFinSearchUnified`

Reply:
@mmachatschek 
Ignore the last comment. We leave this as is.